bundle-workflow/src/test_workflow/config/test_manifest.yml

```diff
@@ -1,13 +1,51 @@
 components:
   - name: index-management
     integ-test:
+      test-configs:
+        - with-security
+        - without-security
+    bwc-test:
+      dependencies:
+      test-configs:
+        - with-security
+        - without-security
+  - name: anomaly-detection
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+    bwc-test:
+      dependencies:
+      test-configs:
+        - with-security
+        - without-security
+  - name: asynchronous-search
+    integ-test:
+      test-configs:
+        - with-security
+        - without-security
+    bwc-test:
+      dependencies:
```

Comment:
*Nit*: This section can be removed as well. You can do it as part of another PR though. Don't want to block the approval for it.

Reply:
If bwc-test doesn't uses `dependencies`. I can remove it